static/app/components/hotkeys.tsx

```diff
@@ -0,0 +1,107 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {getKeyCode} from 'sentry/utils/getKeyCode';
+
+const macModifiers = {
+  18: '⌥',
+  17: '⌃',
+  91: '⌘',
+};
+
+const normalModifiers = {
+  18: 'ALT',
+  17: 'CTRL',
+  91: 'CMD',
+};
+
+const genericGlyphs = {
+  16: '⇧',
+  8: '⌫',
+  37: '←',
+  38: '↑',
+  39: '→',
+  40: '↓',
+  107: '+',
+};
+
+const keyToDisplay = (
+  key: string,
+  isMac: boolean
+): {node: React.ReactNode; specificToOs: 'macos' | 'generic'} => {
+  // Handle escaped + case
+  key = key === '\\+' ? '+' : key;
+
+  let keyStr = key.toUpperCase();
+  let specificToOs: 'macos' | 'generic' = 'generic';
+
+  const keyCode = getKeyCode(key);
+  if (keyCode) {
+    const modifierMap = isMac ? macModifiers : normalModifiers;
+    keyStr = modifierMap[keyCode] ?? genericGlyphs[keyCode] ?? keyStr;
+
+    if (keyStr === 'CMD') {
+      specificToOs = 'macos';
+    }
+  }
+
+  return {node: <Key>{keyStr}</Key>, specificToOs};
+};
+
+type Props = {
+  /**
+   * Pass key combinations in with + as the separator.
+   * For example: command+option+x
+   *
+   * Use comma for fallback key combos when the first one contains a key that does not exist on that os (non-mac):
+   * command+option+x,ctrl+shift+x
+   * (does not have to be the same combo)
+   *
+   * Escape the + key with a slash \+
+   */
+  value: string[] | string;
+  forcePlatform?: 'macos' | 'generic';
+};
+
+const Hotkeys = ({value, forcePlatform}: Props) => {
+  // Split by commas and then split by +, but allow escaped /+
+  const hotkeySets = (Array.isArray(value) ? value : [value]).map(o =>
+    o.trim().split(/(?<!\\)\+/g)
```

Comment:
Careful. I think negative lookbehind like this MIGHT not work in safair

Reply:
let me double check just to make sure

Reply:
Ok it is a problem, you know, might be easier to just ignore the \+ case